appveyor.yml

```diff
@@ -0,0 +1,14 @@
+version: 1.0.{build}
```

Comment:
what's that? and why it is 1.0.X?

Reply:
https://www.appveyor.com/docs/build-configuration/#build-versioning

I believe it can be changed to whatever is desired, even to a tag name when building a release (using APPVEYOR_REPO_TAG_NAME).

My understanding is that release binaries are built outside of CI at present, so the version number above could just as well be ignored.